torch/autograd/__init__.py

```diff
@@ -303,7 +303,7 @@ def variable(*args, **kwargs):
     raise RuntimeError("autograd initialization failed")
 
 # Import all native method/classes
-from torch._C._autograd import (DeviceType, ProfilerActivity, ProfilerState, ProfilerConfig, ProfilerEvent,
+from torch._C._autograd import (DeviceType, ProfilerActivity, ProfilerState, ProfilerConfig, ProfilerEvent, ExperimentalConfig,
```

Comment:
Why do you expose this both on autograd, autograd.profiler and profiler? I would expect that this appears only once on the public API no?

Reply:
yea the way it is done all of these C types are first exposed in autograd/__init__.py 
then you see them getting imported in torch/profiler, the code is similar for ProfilerActivity
```
from torch.autograd import ProfilerActivity, kineto_available, ExperimentalConfig
```

Reply:
sorry let me clarify my comment, you are right that in should be exposed only in public API. I think I followed the current way this is done :

Reply:
> As a result you have to import the C types into autograd init.py
torch/autograd/__init__.py

This is not true AFAIK you can import it anywhere you want!

Reply:
Oh I didn't know that, let me try to import the csrc autograd.profiler and profiler

Reply:
@albanD please let me know if this looks better, thanks.